packages/react-components/src/components/GridLayout.tsx

```diff
@@ -67,10 +67,13 @@ export const GridLayout = (props: GridLayoutProps): JSX.Element => {
   }, [numberOfChildren, currentWidth, currentHeight]);
 
   return (
-    <div ref={containerRef} className={gridLayoutStyle}>
-      <BiGrid gridProps={gridProps} styles={styles}>
-        {children}
-      </BiGrid>
+    <div
+      ref={containerRef}
+      className={mergeStyles(gridLayoutStyle, createGridStyles(numberOfChildren, gridProps), styles?.root, {
+        '> *': styles?.children
```

Comment:
we have our own styles on both sides of the `styles?.root`, when the merge happens - do params on the right hand side take precedence over params to the left if they change the same css property?
If we so can we keep all our styles on the left to ensure contoso can overwrite what they like

Reply:
(and can this be merged into gridLayoutStyles?)

Reply:
Yes, params on the right hand side take precedence. `styles?.root` and `styles?.children` are  GridLayoutStyles which contoso defines to override our styles. I guess it looks weird that I had to separate `styles?.root` and `styles?.children` but I could not think of another notation to have them combined.

Reply:
Oh I totally missed this is passing down contoso styles, your code is correct ignore me! ^_^